resources/sdk/purecloudjava/templates/ApiClient.mustache

```diff
@@ -975,4 +984,48 @@ public class ApiClient implements AutoCloseable {
         @Override
         public void close() throws Exception { }
     }
+
+    private ApiClientConnectorResponse retryInvoke(ApiClientConnectorRequest connectorRequest, ApiClientConnectorResponse connectorResponse) {
+        int retries = 0;
+        long waitTime = 1000L;
+        boolean retry = false;
+        int statusCode;
+        int MAX_RETRIES = 0;
+        long MAX_WAIT_INTERVAL = 32000;
+
+        try {
+            final int retry_count = properties.getProperty(ApiClientConnectorProperty.MAX_RETRIES, Integer.class, 0);
+            if (retry_count > 0) {
+                MAX_RETRIES = retry_count;
+            }
+
+            do {
+                connectorResponse = connector.invoke(connectorRequest);
+                statusCode = connectorResponse.getStatusCode();
+
+                if (statusCode >= 300) {
+                    if (connectorResponse.getHeaders().containsKey("Retry-After")) {
+                        waitTime = Integer.parseInt(connectorResponse.getHeaders().get("Retry-After"));
+                    }
+
+                    if (statusCode == 429) {
+                        retry = true;
+                        Thread.sleep(waitTime * 100L);
```

Comment:
Is this correct? The retry-after header is in seconds and thread.sleep expects miliseconds, which should mean multiplying by 1000, not 100.

Reply:
Technically it should be 1000 but while testing I was getting two-digit number from V2 all the time so to make it a realistic wait time I made it 100. But now making it 1000 again.